core-services/image-mirroring/openshift/mapping_origin_4_10

```diff
@@ -210,6 +210,7 @@ registry.ci.openshift.org/origin/4.10:node-problem-detector-operator quay.io/ope
 registry.ci.openshift.org/origin/4.10:oauth-apiserver quay.io/openshift/origin-oauth-apiserver:4.10 quay.io/openshift/origin-oauth-apiserver:4.10.0 quay.io/openshift/origin-oauth-apiserver:latest
 registry.ci.openshift.org/origin/4.10:oauth-proxy quay.io/openshift/origin-oauth-proxy:4.10 quay.io/openshift/origin-oauth-proxy:4.10.0 quay.io/openshift/origin-oauth-proxy:latest
 registry.ci.openshift.org/origin/4.10:oauth-server quay.io/openshift/origin-oauth-server:4.10 quay.io/openshift/origin-oauth-server:4.10.0 quay.io/openshift/origin-oauth-server:latest
+registry.ci.openshift.org/origin/4.10:oc-mirror quay.io/openshift/origin-oc-mirror:4.10 quay.io/openshift/origin-oc-mirror:4.10.0 quay.io/openshift/origin-oc-mirror:latest
```

Comment:
@smarterclayton i see no problem w/ this name.  In the spirit of you shadowing me on this for a bit, though, are you ok with it?

Reply:
What's `oc mirror`?

Reply:
https://github.com/openshift/oc-mirror

Reply:
plugin for oc that will handle image mirroring:

https://github.com/openshift/oc-mirror

(I am still trying to run down what the future intentions are w/ respect to this plugin vs "oc mirror" itself)

Reply:
It looks distinct enough from `oc image mirror` (raw image mirroring), `oc adm release mirror` (mirroring a single release), and `oc-mirror` (mirror sets of content related to openshift).

As this is our first oc plugin, a part of me struggles between `oc-plugin-mirror` and `oc-mirror`, but since the command is usable on its own, is clearly part of the openshift client ecosystem, and potentially a plugin, `oc-mirror` seems fine.  It does mean we'll need to apply some careful selection to other plugin names so as to ensure they hit the right goals.

Barring any further concerns from Ben, I'm ok with the naming and the pattern it establishes as is.  Ben?

Reply:
yup, i also debated if "oc-plugin-mirror" would be preferred, but the "oc-" namespace seems like that may be sufficient to make that clear.  (oc-rpms being an exception to that logic, at present)

I agree with the "usable on its own" logic